### ISSUE TITLE
The breadcrumb cannot be used on ES5

### ISSUE THREAD

**Issue by dedrazer** [udayvunnam/xng-breadcrumb#65]
## 🐞 Bug report

Using the latest version of this package requires changing the project target version to ES2015.

### Description

Is it possible to change the project target to ES5?

#### 🔬 Minimal Reproduction

yarn the breadcrumb on an ES5 solution.

#### 💻Your Environment

**Angular Version:**

9

**Scully Version:**

N/A

## 🔥 Exception or Error

"ERROR in getInternalNameOfClass() called on a non-ES5 class: expected BreadcrumbItemDirective to have an inner class declaration"

**Comment by udayvunnam** [comment]
@dedrazer does this work with previous version of xng-breadcrum. Please provide last version where you get it to work with ES5

**Comment by dedrazer** [comment]
I tried had tried version 4 and it worked without hassle, but I did not test version 5.

**Comment by dedrazer** [comment]
it seems version 5 is also good

**Comment by dedrazer** [comment]
6.2 works also. The issue started with 6.3

**Comment by udayvunnam** [comment]
With 6.3 we upgraded @angular/cli to version 11. I wonder when there is no breaking change from Angular why this happens.

For a long time, we have been using "target": "es2015". I wonder what's causing an issue now

**Comment by dedrazer** [comment]
The error message points toward BreadcrumbItemDirective, that's all I know.

**Comment by dedrazer** [comment]
do you reckon this will get fixed? we would like to keep using the latest version of xng on our application

**Comment by udayvunnam** [comment]
@dedrazer Thanks for showing effort in keeping the library up-to-date. Have you tried this solution https://stackoverflow.com/a/63727427

**Comment by dedrazer** [comment]
@udayvunnam, yes I have. It works locally but on the CI Pipeline the error is still thrown.

**Comment by dedrazer** [comment]
I wouldn't like to be bothersome, it would be really nice to have a solution for this on ES5. In 6.2 I cannot get the @alias working for some reason and I cannot access the changes I made in 6.4.0.

**Comment by udayvunnam** [comment]
@dedrazer This might be a minor TS target version change. I will check this soon.

**Comment by dedrazer** [comment]
thanks @udayvunnam, I appreciate it!

**Comment by udayvunnam** [comment]
@dedrazer compiling the library with typescript target of es5 increases library size significantly (almost double). 

Can you please try moving your project version to es2015 to make this work, else please consider upgrading the package whenever you update the version.

**Update tsconfig.json**

```
compilerOptions : {
	"target": "es5"
}
```

**Comment by udayvunnam** [comment]
Not stale
